packages/sysweb3-keyring/src/keyring-manager.ts

```diff
@@ -759,6 +729,46 @@ export const KeyringManager = (): IKeyringManager => {
     };
   };
 
+  // const { wallet: _wallet } = getDecryptedVault();
```

Comment:
Validate if the commented code is indeed useless and won't be necessary any longer and if so remove it